CHANGELOG.md

```diff
@@ -22,7 +22,7 @@
             * Always - output methods **never** strip ANSI escape sequences, regardless of the output destination
             * Never - output methods strip all ANSI escape sequences
     * Added ``macro`` command to create macros, which are similar to aliases, but can take arguments when called
-    * ``alias`` is now an argparse command with subcommands to create, list, and delete aliases 
```

Comment:
Is this a merge mistake?

Reply:
I view this as a fix because there were duplicated lines for this on master - one here and one below under the Deletions (possibly breaking changes) section.